Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:20.11-alpine3.19
+FROM node:20.12.1-alpine3.19
```

Comment:
Please remove the patch version so containers will have the most recent patches when pulled.